noir/noir-repo/compiler/noirc_frontend/src/hir/type_check/mod.rs

```diff
@@ -624,25 +639,26 @@ mod test {
     }
 
     fn type_check_src_code(src: &str, func_namespace: Vec<String>) {
-        type_check_src_code_errors_expected(src, 0, func_namespace);
+        type_check_src_code_errors_expected(src, 0, func_namespace, 0);
     }
 
     // This function assumes that there is only one function and this is the
     // func id that is returned
     fn type_check_src_code_errors_expected(
         src: &str,
-        expected_number_errors: usize,
+        expected_num_parser_errs: usize,
```

Comment:
I don't think we need this parameter - it should always be zero.

Reply:
good call I assumed some of the other tests were using it. I removed it, must have been outdated.